content/applications/hr/fleet/new_vehicle.rst

```diff
@@ -36,84 +49,248 @@ This section of the vehicle form relates to the person who is currently driving
 any plans for a change in the driver in the future, and when.
 
 - :guilabel:`Driver`: select the driver from the drop-down menu, or type in a new driver and click
-  either :guilabel:`Create` or :guilabel:`Create and Edit`.
+  either :guilabel:`Create "driver"` or :guilabel:`Create and edit...` to :ref:`create a new driver
+  and edit the driver details <fleet/new_vehicle/add-driver>`.
+
+  .. important::
+     A driver does **not** have to be an employee. All drivers are stored within the *Fleet*
+     application, and if the *Contacts* application is installed, they are also stored there.
```

Comment:
Same- good point! I had to test this a few times to make sure I understood this as well, since it's really odd that I couldn't access specific lists within an app.